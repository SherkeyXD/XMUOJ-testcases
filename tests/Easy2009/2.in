4754 855
4707 392
9130 516
9995 603
473 255
7488 761
5520 511
8274 825
2894 582
7095 210
1034 376
776 726
2655 189
2170 422
