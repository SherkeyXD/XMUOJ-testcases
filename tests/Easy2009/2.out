5686.41
5176.04
9751.85
10708.55
753.45
8345.25
6114.16
9199.97
3537.03
7398.79
1447.22
1534.39
2902.56
2645.16
